common/djangoapps/enrollment/views.py

```diff
@@ -411,18 +411,29 @@ def post(self, request):
             # have to check if it is inactive or not, because if it is, we are still upgrading if the mode is different,
             # and either path will re-activate the enrollment.
             #
-            # Only server-to-server calls will currently be allowed to modify the mode for existing enrollments. All
-            # other requests will go through add_enrollment(), which will allow creating of new enrollments, and
-            # re-activating enrollments
+            # Only server-to-server calls are currently allowed to deactivate or modify the mode of existing
```

Comment:
Updated. I've moved half of this to the method docstring and made the remaining half more concise in addition to moving it to the relevant line.